pages/fleet/[id].vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div>
+    <div class="mb-10 mt-7 text-xs">
+      Category / <span class="font-semibold">{{ zeppelin.name }}</span>
+    </div>
+    <div class="grid grid-cols-2 gap-x-16 gap-y-5 mb-44">
+      <img
+        :src="zeppelin.image.path"
+        :alt="zeppelin.image.alt"
+        class="w-full rounded"
+      />
+      <div class="w-full">
+        <h1 class="mb-2">{{ zeppelin.name }}</h1>
+        <h2 class="mb-2 font-normal">
+          From <span class="font-bold">${{ zeppelin.lowestPrice }}</span> per
+          day
+        </h2>
+        <hr class="border-gray-300 border mb-5" />
+        <p class="mb-5">
+          Description of this fantastic flying boi. It’s a great one!
+        </p>
+        <button class="bg-blue-200 text-white px-5 py-2 rounded-md mb-14">
+          Book
+        </button>
+        <BaseCollapsableItem
```

Comment:
as they are elements grouped as a list you can wrap them in `ul` and `li` elements 👌